doc/api/errors.md

```diff
@@ -1463,6 +1463,12 @@ The V8 platform used by this instance of Node.js does not support creating
 Workers. This is caused by lack of embedder support for Workers. In particular,
 this error will not occur with standard builds of Node.js.
 
+<a id="ERR_MODULE_NOT_FOUND"></a>
```

Comment:
Resolver errors always have code "MODULE_NOT_FOUND". Changing this to "ERR_MODULE_NOT_FOUND" just for one specific case seems both inconsistent and breaking to me.

For example, lots of code has guards that do `try { require.resolve(x) } catch (e) { if (e.code === 'MODULE_NOT_FOUND' }` sort of thing.

Reply:
I wondered why we did not yet change that error code but that makes sense. Is this something we ever plan to fix (as in: align with all other error codes in Node.js)?

Reply:
I changed it to `MODULE_NOT_FOUND`.